src/components/common/CreateUserWizard/StepUserEmail.tsx

```diff
@@ -0,0 +1,208 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { Checkbox, Form, Input } from '~shared/Fields';
+import { WizardStepProps } from '~shared/Wizard';
+import QuestionMarkTooltip from '~shared/QuestionMarkTooltip';
+import { Message } from '~types';
+
+import {
+  FormValues,
+  UserStepTemplate,
+  ContinueWizard,
+} from '../CreateUserWizard';
+import { stepUserEmailValidationSchema as validationSchema } from './CreateUserWizardValidation';
+
+import styles from './StepUserEmail.css';
+
+const displayName = 'common.CreateUserWizard.StepUserEmail';
+
+const MSG = defineMessages({
+  heading: {
+    id: `${displayName}.heading`,
+    defaultMessage: "Welcome to Colony, let's get started!",
+  },
+  description: {
+    id: `${displayName}.description`,
+    defaultMessage:
+      'Enter your email address below to enhance your experience.',
+  },
+  label: {
+    id: `${displayName}.label`,
+    defaultMessage: 'Your email',
+  },
+  explanation: {
+    id: `${displayName}.explanation`,
+    defaultMessage: 'We use your email address to:',
+  },
+  confirmHumanity: {
+    id: `${displayName}.confirmHumanity`,
+    defaultMessage: "Confirm you're human so we can pay your gas costs 🤑",
+  },
+  confirmHumanityTooltip: {
+    id: `${displayName}.confirmHumanityTooltip`,
+    defaultMessage: `"Gas" is the term for payment required to interact with the blockchain. To keep things simple, we want to cover these costs for you.`,
+  },
+  sendNotifications: {
+    id: `${displayName}.sendNotifications`,
+    defaultMessage: 'Send notifications about activity in your colony 📬',
+  },
+  sendNotificationsTooltip: {
+    id: `${displayName}.sendNotificationsTooltip`,
+    defaultMessage: `We want to ensure that you are notified when things happen in your DAO that are important to you, e.g. a vote starts.`,
+  },
+});
+
+enum EmailPermissions {
+  isHuman = 'isHuman',
+  sendNotifications = 'sendNotifications',
+}
+
+type Permission = `${EmailPermissions}`;
+
+class PermissionDescription {
```

Comment:
Using a class reduces the boilerplate of creating object literals, especially given I can't use property value shorthand here. Probably not that noticeable here given there are only two items in the array, but still strikes me as "correct".

Reply:
I have nothing against this pattern but the file has grown a bit large, maybe it's worth extracting the class to a separate file?

Reply:
Agree with Jakub, should go in a separate file (inside the current Component's folder)